### PR TITLE
Feat/877 Transaction button updates

### LINKED ISSUE #877: Follow up for TransactionButton styles and error state
- Error state below button
- Disabled styles for button (cursor: not-allowed/default and border, muted text color)

### PR DESCRIPTION
**Pull request by mattrussell36**
Closes #877

### CODE REVIEW

File: src/components/transaction-button/transaction-button.scss [hunk 1]
```diff
@@ -3,15 +3,35 @@
...
     justify-content: space-between;
     align-items: flex-start;
-    margin: 5px 0 0 0;
-    font-size: 14px;
   }
 
-  // Complete and Error states
+  &__message {
+    padding: 5px 0 5px 7px;
+    border-left: 3px solid #ff7a1a;
```
Comment (pgriscti) [inline src/components/transaction-button/transaction-button.scss:24]: I think we should add these colours to our colors config

File: src/components/transaction-button/transaction-button.tsx [hunk 2]
```diff
@@ -114,8 +99,51 @@ export const TransactionButton = ({
...
+    );
+  }
+
+  if (txHash) {
+    return (
+      <div className="transaction-button__footer">
+        <p className="transaction-button__txhash">
+          <span>{t("transaction")}</span>
+          <a
+            href={`https://ropsten.etherscan.io/tx/${txHash}`}
```
Comment (pgriscti) [inline src/components/transaction-button/transaction-button.tsx:137]: why is this hard-coded to ropsten?

File: src/i18n/translations/dev.json [hunk 3]
```diff
@@ -434,7 +434,8 @@
   "withdrawFormSubmitButtonPending": "Preparing",
   "withdrawalsTitle": "Incomplete withdrawals",
   "withdrawalsText": "These withdrawals need to be completed with an Ethereum transaction.",
-  "withdrawalsNone:": "You dont have any pending withdrawals.",
+  "withdrawalsNone": "You dont have any pending withdrawals.",
```
Comment (pgriscti) [inline src/i18n/translations/dev.json:437]: dont = don't

File: src/i18n/translations/dev.json [hunk 4]
```diff
@@ -469,5 +470,6 @@
   "vegaWallet": "Vega Wallet",
   "rewardsComingSoon": "Rewards is coming soon",
   "withdrawsComingSoon": "Withdraws are coming soon",
-  "associationChoice": "You have $VEGA tokens held by the vesting contract. Would you like to associate these or $VEGA directly from wallet?"
+  "associationChoice": "You have $VEGA tokens held by the vesting contract. Would you like to associate these or $VEGA directly from wallet?",
```
Comment (pgriscti) [inline src/i18n/translations/dev.json:473]: Grammatical error? "Would you like to associate these or $VEGA directly from wallet?"

**Review by dexturr — APPROVED** [review]

**Review by pgriscti — APPROVED** [review]

**Pull request merged**